docs/components/plugins/APISection.tsx

```diff
@@ -36,6 +36,8 @@ const isHook = ({ name }: GeneratedData) =>
 
 const isListener = ({ name }: GeneratedData) => name.endsWith('Listener');
 
+const isProp = ({ name }: GeneratedData) => name.includes('Props') && name !== 'ErrorRecoveryProps';
```

Comment:
I'm a bit confused here, the function name suggest checking for `"prop"` string and we're given checking for string `"Props"` and with one exception 🤔
I've got no suggestion here, but please add some comments in the placed you call this function ok? 😉

Reply:
This is the same issue as with `useSystemBrightnessAsync` which violates hooks naming convention.

<img width="592" alt="Screenshot 2021-08-03 at 14 07 51" src="https://user-images.githubusercontent.com/719641/128012864-46168c1d-c6c6-449b-b227-4c1ad1036efe.png">

As you can see I'm leaving the comments usually, but I would like to see the rename/refactor of those kind of methods. I have tried to do this in: https://github.com/expo/expo/pull/13693#pullrequestreview-710411962, but we unfortunately need to introduce the depreciation period for them first.

Also I'm not sure what exactly confuses you in the Prop case, it would be nice if you can explain be the issue, maybe I would be able to write the code more clear. 🙂

Reply:
Well, I'm not this familiar with the code and, especially when I only read part of it, I'm feeling unsure what's being processed there 😉 While I was reading the code I was lacking the context and understanding about what's being extracted there, but now I do understand (because I read a little more and looked again at docs pages 😅). Sorry for the confusion.
Thank you for being careful and aiming for a bette/more generic codebase eventually! 👏